standalone/src/models/cart/cart/cart/constants.ts

```diff
@@ -11,7 +11,7 @@ export const taxMode = {
   Disabled: 'Disabled',
 } as const;
 
-export const taxRoundingMode = {
+export const roundingMode = {
```

Comment:
Why do we need to change this name?
The constants are exported in the models so this would break consumers which are using this one.